oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/type/Process.java

```diff
@@ -44,6 +44,8 @@ public class Process {
     private String agentId;
     @Setter
     private String detectType;
+    @Setter
+    private String profilingSupportStatus;
```

Comment:
What are the statuses?

Reply:
The value is the name of the `ProfilingSupportStatus` enum.